hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecord.java

```diff
@@ -169,15 +179,17 @@ public HoodieOperation getOperation() {
     return operation;
   }
 
+  public Comparable<?> getOrderingValue() {
+    return orderingVal;
+  }
+
   public T getData() {
     if (data == null) {
-      throw new IllegalStateException("Payload already deflated for record.");
+      throw new IllegalStateException("HoodieRecord already deflated for record.");
     }
```

Comment:
Why this change ? Please revert it back !!!

Reply:
I think this was accidental find-replace kind of change

Reply:
done